### PR TITLE
Update bindgen.

### PR DESCRIPTION
**Pull request by emilio**

### CODE REVIEW

**Comment by emilio** [comment]
Windows CI failure looks unrelated.

File: lmdb-sys/src/bindings.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub const MDB_BAD_VALSIZE: ::libc::c_int = -30781;
 pub const MDB_BAD_DBI: ::libc::c_int = -30780;
 pub const MDB_LAST_ERRCODE: ::libc::c_int = -30780;
 pub type mdb_mode_t = mode_t;
-pub type mdb_filehandle_t = filehandle_t;
+pub type mdb_filehandle_t = ::libc::c_int;
```
Comment (Kerollmops) [inline lmdb-sys/src/bindings.rs]: The windows CI failures are related to this change. On windows the file descriptors are `void pointers` and I specified that [in the lib.rs file under the type `filehandle_t`](https://github.com/mozilla/lmdb-rs/blob/f0fa429eaa2f3c5e4ff87cc0dd04d98c248a1ef7/lmdb-sys/src/lib.rs#L18-L20).

https://dev.azure.com/mykzilla/lmdb-rkv/_build/results?buildId=369&view=logs&j=57b2b475-8d2a-5b04-5d48-0e52fa3a6ad3&t=8b5e7da2-0c8d-5281-46c0-54f8db5b4c9f&l=83

Comment (Kerollmops) [reply lmdb-sys/src/bindings.rs]: Ho, you fixed that in #77, that's nice.

Comment (emilio) [reply lmdb-sys/src/bindings.rs]: Yeah, that shouldn't have been a manual change, otherwise it risks getting overridden :/

**Pull request merged**